### PR TITLE
Fix re-render error with profile image

### PR DESCRIPTION
**Pull request by kip-west**
#### What's this PR do?
- Fixes error with profile image re-render
- Adds UPDATE_USER action and case to reducer file
#### Where should the reviewer start?
- The bulk of this work happens in `user/api/actions` & `user/api/state`; check out the new action and reducer here. Then head to the ProfilePicture component to see this method getting called in `saveChanges`
#### How should this be manually tested?
- Open app in the browser, visit a user's profile page. Ensure that you can upload an image, that it re-renders, and persists over a page reload/switching pages.
#### Any background context you want to provide?
- After getting the initial profile photo upload setup, we needed a way to update `store` automatically and re-render the image.
#### What are the relevant tickets?
- #32

### CODE REVIEW

File: code/web/src/modules/user/ProfilePicture.js [hunk 1]
```diff
@@ -85,4 +86,4 @@ function profileState(state) {
     user: state.user,
   }
 }
-export default connect(profileState, { upload, messageShow, messageHide, updateUser })(ProfilePicture)
\ No newline at end of file
+export default connect(profileState, { upload, messageShow, messageHide, updateUser, getUser })(ProfilePicture)
```
Comment (kathytbui) [inline code/web/src/modules/user/ProfilePicture.js:89]: Is this so that we can retrieve the use again after we updated them?

**Review by kathytbui — APPROVED** [review]

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -137,4 +138,25 @@ export function updateUser(user, id) {
...
   }
+}  
+// Get single user
+export function getUser(id) {
+  return dispatch => {
+    axios.post(routeApi, query({
+      operation: 'user',
+      variables: { id },
+      fields: ['name', 'email', 'role', 'image', 'id']
+    }))
```
Comment (carlymclift) [inline code/web/src/modules/user/api/actions.js:149]: really great catch on here

**Review by carlymclift — APPROVED** [review]
I'm really glad we got this working!

**Review by dtsehl — APPROVED** [review]

**Comment by leighlars** [comment]
Great job, Jake. I will rebase this.

File: code/web/src/modules/user/ProfilePicture.js (hunk 1 above)
Comment (kip-west) [reply code/web/src/modules/user/ProfilePicture.js:89]: It is!! This method is from our `actions.js` file. The `connect` function here links our component `ProfilePicture` to `profileState` & all of the methods from `{ upload, ..., getUser }`- which is the way we link any component to Redux functionality.

**Pull request merged**